ui-spacetimechart/src/utils/canvas.ts

```diff
@@ -139,8 +174,13 @@ export function drawAliasedDisc(
   { x: centerX, y: centerY }: Point,
   radius: number,
   [r, g, b]: RGBColor | RGBAColor,
-  drawOnBottom: boolean
+  drawOnBottom: boolean,
+  scalingRatio: number = 1
 ): void {
+  centerX = Math.round(centerX * scalingRatio);
+  centerY = Math.round(centerY * scalingRatio);
+  radius = Math.round(radius * scalingRatio);
```

Comment:
Old code was using `Math.ceil` here, do we want to retain the old behavior?

I don't have the full context, but maybe it's to avoid ending up with `radius = 0`?